ugandaemr-304.md

```diff
@@ -0,0 +1,58 @@
+## UgandaEMR 3.0.4 Release Notes
+
+Currently, work in progress
+
+### New Features
+1.  Patient Flags
+    * Due for third DNA PCR
+    * Over Due for third DNS PCR
+    * Updated Criteria for Due and Over Due for Second DNA PCR
+ 2. HIV MER Indicator Reports 
+    * TX_ML Report
+ 3. Data Extraction Reports
+    * Postnantal Register
+    
+    
+  
+### Fixes Made
+
+       1. Disabled the creation of Multiple encounters on the same visit date
+       2. Reports Module does not start even after starting the module
+       3. EID Card does not allow future dates for Next Appointment Date
+       4. System does not support to measure the number of patients seen by a perticular provider
+       5.Clinical Assessment Forms are innaccessible from the HTML form entry section
+       6. Duplicate Clients in queue
+
+### Links to download files
+
+#### New Installation
+
+For new machines with no UgandaEMR installed
+
+* 32-bit installer -https://sourceforge.net/projects/ugandaemr/files/3.0.4/ugandaemr3-0-4-installer-32bit.exe/download)
+* 64- bit installer -https://sourceforge.net/projects/ugandaemr/files/3.0.4/ugandaemr3-0-4-installer-64bit.exe/download)
+
+#### WAR File
+
+[https://sourceforge.net/projects/ugandaemr/files/3.0.4/openmrs.war/download](https://sourceforge.net/projects/ugandaemr/files/3.0.4/openmrs.war/download)
+
+#### Upgrade Installer
+
+**For existing versions of UgandaEMR 2.0.\* to 3.0.4**
```

Comment:
@solemabrothers what is the \* for?

Reply:
@ssmusoke we have two different versions i.e, 2.0.0 and 2.0.1, the * is to represent any of the versions

Reply:
Use 2.0.x, similar to what we use below